src/wccp2.cc

```diff
@@ -1105,6 +1100,29 @@ wccp2ConnectionClose(void)
     wccp2_connected = 0;
 }
 
+class Wccp2Rr : public RegisteredRunner
+{
+public:
+    virtual void useConfig() override {
+        if (IamPrimaryProcess()) {
+            wccp2Init();
+            wccp2ConnectionOpen();
+        }
+    }
+
+    // reconfigure fully shuts down, then restarts WCCP
+    virtual void startReconfigure() override { startShutdown(); }
+    virtual void syncConfig() override { useConfig(); }
+
+    // must run before comm_exit()
```

Comment:
Using same solution as suggested for wccp.cc :
```suggestion
    // we use startShutdown() instead of later events because 
    // Comm socket closures require main loop iterations
```